lib/bolt/bolt_option_parser.rb

```diff
@@ -181,334 +193,381 @@ module            Manage Bolt project modules
           secret            Create encryption keys and encrypt and decrypt values
           task              Show and run Bolt tasks
 
-      GUIDES
+      #{colorize(:cyan, 'GUIDES')}
           For a list of guides on Bolt's concepts and features, run 'bolt guide'.
+          Find Bolt's documentation at https://bolt.guide.
     HELP
 
     APPLY_HELP = <<~HELP
-      NAME
+      #{colorize(:cyan, 'NAME')}
           apply
 
-      USAGE
+      #{colorize(:cyan, 'USAGE')}
           bolt apply [manifest] {--targets TARGETS | --query QUERY | --rerun FILTER}
             [options]
 
-      DESCRIPTION
+      #{colorize(:cyan, 'DESCRIPTION')}
           Apply Puppet manifest code on the specified targets.
 
-      EXAMPLES
+      #{colorize(:cyan, 'DOCS')}
```

Comment:
```suggestion
      #{colorize(:cyan, 'Documentation')}
```

Prefer spelling the entire word out.